Index: glib2.0/gio/tests/meson.build
===================================================================
--- glib2.0.orig/gio/tests/meson.build
+++ glib2.0/gio/tests/meson.build
@@ -262,7 +262,10 @@ if host_machine.system() != 'windows'
     },
     'gdbus-peer-object-manager' : {},
     'gdbus-sasl' : {},
-    'live-g-file' : {},
+    'live-g-file' : {
+      # FIXME: Fails in Pexip build environment
+      'can_fail': true,
+    },
     'portal-support-flatpak-none' : {
       'extra_sources': ['../gportalsupport.c', '../gsandbox.c', 'portal-support-utils.c'],
       'c_args': ['-DG_PORTAL_SUPPORT_TEST'],
Index: glib2.0/glib/tests/meson.build
===================================================================
--- glib2.0.orig/glib/tests/meson.build
+++ glib2.0/glib/tests/meson.build
@@ -180,7 +180,10 @@ glib_tests = {
   'utils' : {
     'c_standards': c_standards.keys(),
   },
-  'utils-isolated' : {},
+  'utils-isolated' : {
+    # FIXME: Fails in Pexip build environment
+    'can_fail': true,
+  },
   'unicode' : {},
   'unicode-encoding' : {},
   'unicode-normalize': {},
